modules/backup-and-restore/pages/differential-backups.adoc

```diff
@@ -0,0 +1,91 @@
+= Differential Backup
+
+== Terminology Definition
+Full backups comprise entire data backup sets, regardless of already existing backups or data change circumstances.
+
+* Differential backups comprise data files that have changed since the most recently completed full backup.
```

Comment:
Probably no "files" here, differential backups backup the kafka messages, not database files.

Reply:
"most recently completed" change to "most recently completed or most recently restored".
For example, if you have full backup A, full backup B, A happened before B. If you restore A, then the next differential backup is based on A, instead of B, although B was completed more recently.

Reply:
Thank you I changed these with latest commit